kroxylicious-test-tools/src/main/java/io/kroxylicious/test/tester/KroxyliciousTesters.java

```diff
@@ -49,6 +61,8 @@ public static KroxyliciousTester kroxyliciousTester(ConfigurationBuilder builder
      * @return KroxyliciousTester
      */
     public static MockServerKroxyliciousTester mockKafkaKroxyliciousTester(Function<String, ConfigurationBuilder> configurationForMockBootstrap) {
-        return new MockServerKroxyliciousTester(MockServer.startOnRandomPort(), configurationForMockBootstrap);
+        return new MockServerKroxyliciousTester(MockServer.startOnRandomPort(), configurationForMockBootstrap, DefaultKroxyliciousTester::spawnProxy,
```

Comment:
It's a bit odd that that you use the KroxyliciousTesterBuilder() to create some testers (default) but not others (mock).

Reply:
It is, however the mock doesn't play nicely with the tester as things stand. I started down the path of using the builder for the mock but that was quite a big change from what we have here see https://github.com/kroxylicious/kroxylicious/pull/661. So I split them to try and keep the reviews manageable.